plugins/modules/mso_tenant_site.py

```diff
@@ -261,6 +261,8 @@ def main():
     # Get tenant_id and site_id
     tenant_id = mso.lookup_tenant(module.params.get("tenant"))
     site_id = mso.lookup_site(module.params.get("site"))
+    if not site_id:
```

Comment:
What would happen on the present/query states? Should we fail or just exit?

Reply:
If the site_id is an empty list of None, then the task will simply exit with an empty response(the status will be passed with an empty result).